nixos/modules/system/boot/networkd.nix

```diff
@@ -67,12 +83,37 @@ let
         (assertInt "CombinedChannels")
         (assertRange "CombinedChannels" 1 4294967295)
         (assertInt "RxBufferSize")
+        (assertInt "RxMiniBufferSize")
+        (assertInt "RxJumboBufferSize")
         (assertInt "TxBufferSize")
+        (assertValueOneOf "RxFlowControl" boolValues)
+        (assertValueOneOf "AutoNegotiationFlowControl" boolValues)
+        (assertByteFormat "GenericSegmentOffloadMaxBytes")
```

Comment:
can this one have a range check too? if not it's probably fine to make it a plain int.

Reply:
The docy say the following for `GenericSegmentOffloadMaxBytes`:
> Specifies the maximum size of a Generic Segment Offload (GSO) packet the device should accept. 
> The usual suffixes K, M, G are supported and are understood to the base of 1024. An unsigned integer in the range 1...65536. Defaults to unset.

`assertByteFormat` is the following check:

https://github.com/NixOS/nixpkgs/blob/5758299f95a26f6c28d7fef36de81ba70716ddb1/nixos/lib/systemd-lib.nix#L39-L49

I mainly used this check because it's also what was used for the other `Bytes` parameters like `MTUBytes`.